client/common/configuration/register.ts

```diff
@@ -1,6 +1,7 @@
 import type { RegisterClientOptions } from '@peertube/peertube-types/client'
 import { renderConfigurationHome } from './templates/home'
-import { renderConfigurationChannel } from './templates/channel'
+import './templates/ChannelConfigurationElement'
```

Comment:
Usually filenames are kebab-case in this project. Should be `templates/channel-configuration-element`.
Maybe we can remove the `element` part. Are there anything else than LitElements in the `templates` folder?

Reply:
No you're right, we can go for the element tag basically. Only case that would get in the way is services that have common names. We could separate them in folders.

Reply:
Yes!